itest/lnd_multi-hop-payments_test.go

```diff
@@ -85,9 +85,19 @@ func testMultiHopPayments(ht *lntest.HarnessTest) {
 		aliceInboundFeeRate     = -50000 // 5%
 	)
 
+	// We update the channel twice. The first time we set the inbound fee,
```

Comment:
I think this behavior is already tested in `req.InboundFee = nil`, plus this is an MPP-related test and should not test policy updates.